source/getting-started/tutorials/sensors/index.md

```diff
@@ -0,0 +1,283 @@
+# Tutorial: Sensor Configuration, Feedback, & Profiling
+
+- **Goal:** Learn how to use the AMDC Timing Manager to get feedback from sensors
+- **Complexity:** 4 / 5
+- **Estimated Time:** TODO
+
+
+## Tutorial Requirements
+
+1. Working AMDC Hardware
+2. Completion of the [Hardware Commands](/getting-started/tutorials/hw-commands/index.md) tutorial
+3. Completion of the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial
+4. Completion of the [Profiling Tasks](/getting-started/tutorials/profiling-tasks/index.md) tutorial
+
+This tutorial expands on the code created in the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial, and uses commands created in the [Profiling Tasks](/getting-started/tutorials/profiling-tasks/index.md) tutorial. Both must be completed before this tutorial.
+
+## Introduction 
+
+In motor control applications, it is important to have consistent timing of both sensor data acquisition and the running of control tasks. In default configuration, the AMDC allows for some jitter between sensor measurement time and control loop timing. This tutorial shows how the AMDC can be configured to elimiate this jitter.
+
+This tutorial provides the source code for:
+* experimenting the effects of the timing manager using the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) app
+
+## Scheduling and Synchronizing:
+
+The AMDC synchronizes running tasks and sensor collection to the PWM carrier wave. Every X PWM periods (where X is set by the function timing_manager_set_ratio()), the AMDC will collect data from sensors. In Legacy mode, the AMDC will run control tasks concurrently with sensor collection. In Post-sensor mode the AMDC will not run control tasks until the sensor collection is complete. 
```

Comment:
"sensor data collection and task execution"

For functions/code snippets, please use the markdown code formatting with the backtick character:

Raw markdown: \`timing_manager_set_ratio()\`

Renders as: `timing_manager_set_ratio()`

"By default, the Timing Manager is set to *Legacy Mode*, in which the AMDC will run control tasks concurrently with sensor data collection. Alternatively, when configured in *Post-Sensor Mode*, the Timing Manager guaruntees that in each control cycle, the AMDC will wait to execute control tasks until the sensor data collection is complete."